.gitignore

```diff
@@ -14,6 +14,7 @@ config.sub
 configure
 depcomp
 .deps/
+fontutils/xrdp-dumpfv1
```

Comment:
Also need fontutils/xrdp-mkfv1 which is only generated if built with `--with-freetype2`